DEVELOPER_SETUP.md

```diff
@@ -0,0 +1,415 @@
+cd src/VirtoCommerce.Platform.Web
+```
+
+2. **Setup SQL Server:**
+
+**Option A: Docker (Recommended)**
+```bash
+docker run -d --name vc-sqlserver \
+  -e "ACCEPT_EULA=Y" \
+  -e "MSSQL_PID=Express" \
+  -e "SA_PASSWORD=Pass@word" \
+  -p 1433:1433 \
+  mcr.microsoft.com/mssql/server:latest
+```
+
+**Option B: Local SQL Server**
+Update `appsettings.Development.json`:
+```json
+{
+  "ConnectionStrings": {
+    "VirtoCommerce": "Data Source=localhost,1433;Initial Catalog=VirtoCommerce3;User ID=sa;Password=Pass@word;TrustServerCertificate=True;"
+  }
+}
+```
+
+3. **Build the project:**
+```bash
+# From root directory
+dotnet build VirtoCommerce.Platform.sln
+```
+
+4. **Run the backend:**
+```bash
+# ⚠️ IMPORTANT: Must be run from src/VirtoCommerce.Platform.Web directory
+cd src/VirtoCommerce.Platform.Web  # If not already there
+dotnet run --environment Development
+
+# Alternative: Run with project path from any directory
+dotnet run --project "src\VirtoCommerce.Platform.Web\VirtoCommerce.Platform.Web.csproj" --environment Development
+```
+
+5. **Verify backend is running:**
+- URL: http://localhost:10645
+- Login: `admin` / `store`
+- You should see the admin dashboard
+
+### Frontend Setup
+
+1. **Clone frontend repository:**
+```bash
+# In a separate directory
+git clone https://github.com/VirtoCommerce/vc-frontend.git
+cd vc-frontend
+```
+
+2. **Configure backend connection:**
+```bash
+# Create .env file with correct backend URL
+echo APP_BACKEND_URL=http://localhost:10645 > .env
+```
+
+3. **Handle Yarn version conflicts:**
+
+The frontend requires Yarn 4.7.0, but most systems have Yarn 1.x. Here are solutions:
+
+**Option A: Enable Corepack (Requires Admin Privileges)**
+```bash
+corepack enable
+npm run dev
+```
+
+**Option B: Use npm instead (If Corepack fails)**
+```bash
+npm install
+npx vite --host 0.0.0.0 --port 3000
+```
+
+**Option C: Bypass yarn precheck**
+```bash
+npx vite --host 0.0.0.0 --port 3000 --force
+```
+
+4. **Start frontend (Handle Process Instability):**
+
+The frontend process can be unstable in some environments. Use this approach:
+
+```powershell
+# Start in separate PowerShell window to prevent crashes
+Start-Process powershell -ArgumentList "-NoExit", "-Command", "cd '$(pwd)'; npx vite --host 0.0.0.0 --port 3000"
+```
+
+5. **Access storefront:**
+- URL: https://localhost:3000
+- ⚠️ **Certificate Warning**: Click "Advanced" → "Proceed to localhost (unsafe)"
+- This is normal for self-signed HTTPS certificates in development
+
+## 🔧 Troubleshooting
+
+### Backend Issues
+
+**Error**: "Unable to run your project... OutputType is 'DockerCompose'"
+```bash
+# You're in the wrong directory! Navigate to the web project:
+cd src/VirtoCommerce.Platform.Web
+dotnet run --environment Development
+```
+
+**Error**: "Address already in use" on port 10645
+```bash
+# Find and kill the process using the port
+netstat -ano | findstr ":10645"
+Stop-Process -Id <ProcessId> -Force
```

Comment:
`Stop-Process` is a PowerShell command, so probably you should replace `bash` with `powershell` everywhere